plugins/grpc/stats/types.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package stats
+
+import (
+	"log"
+	"time"
+
+	istats "github.com/census-instrumentation/opencensus-go/stats"
+	"github.com/census-instrumentation/opencensus-go/tags"
+)
+
+// tagsKey is the metadata key used to identify both the stats tags in the GRPC
+// context metadata, as well as the RpcServerStats info sent back from the
+// server to the client in the GRPC context metadata.
+const tagsKey = "grpc-tags-bin"
```

Comment:
Metadata keys start with `grpc-` are actually reserved, so we should avoid having a `grpc-` prefixed key explicitly in the code...

To make this work, we added functions to set/get trace info: https://github.com/grpc/grpc-go/blob/master/stats/stats.go#L269.

Please use the new functions instead.